src/components/Form/Suggestions/Suggestions.jsx

```diff
@@ -1,60 +1,65 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import { newGuid } from '../ValidationElement'
 import Modal from '../Modal'
 
 export default class Suggestions extends React.Component {
-  constructor(props) {
-    super(props)
-    this.dismissSuggestions = this.dismissSuggestions.bind(this)
-  }
-
   /**
    * Use a suggestion given.
    */
-  useSuggestion(suggestion) {
-    this.props.onSuggestion(suggestion)
+  useSuggestion = (suggestion) => {
+    const { onSuggestion } = this.props
+    onSuggestion(suggestion)
   }
 
   /**
    * This allows the user to bypass the suggestions and add something else
    * we have never seen before.
    */
-  dismissSuggestions(action = 'dismiss') {
-    this.props.onDismiss(action)
+  dismissSuggestions = (action = 'dismiss') => {
+    const { onDismiss } = this.props
+    onDismiss(action)
   }
 
   /**
    * Return the possible suggestions or an empty value if there is nothing to present.
    */
-  suggestions() {
-    return this.props.suggestions.map(suggestion => {
-      return (
-        <div className="suggestion" key={newGuid()}>
-          <div className="value">
-            <h5>{this.props.suggestionLabel}</h5>
-            {this.props.renderSuggestion(suggestion)}
-          </div>
-          <div className="action">
-            <button
-              className="suggestion-btn"
-              onClick={this.useSuggestion.bind(this, suggestion)}>
-              <span>{this.props.suggestionUseLabel}</span>
-              <i className="fa fa-arrow-circle-right" />
-            </button>
-          </div>
+  suggestions = () => {
+    const {
+      suggestions, suggestionLabel, renderSuggestion, suggestionUseLabel,
+    } = this.props
+    return suggestions.map((suggestion, i) => (
+      <div
+        className="suggestion" key={`suggestion-${i}`}
```

Comment:
I don't love that I'm using the index of the current element in the `.map`, but I do think it's a step up from the completely random "guid" that's getting generated. I don't see another unique identifier I could use.